File: C_Programming/Assignment/POC/factorialfun.c
#include<stdio.h>
void factorial();
void main()
{
    factorial();
}
void factorial()
{
    int num,fact=1;
    printf("Enter the value of no.:");
    scanf("%d",&num);
    int temp=num;
    while(num)
    {
        fact=fact*num;
        num--;

    }
    printf("factorial of no:%d is %d",temp,fact);
}

//output:Enter the value of no.:5
//factorial of no:5 is 120

File: Java_Programs/Assignment5Type(3)/Factorialfun.java
package Assignment5Type3;

public class Factorialfun {

	public static void main(String[] args) {
		 int num=5;
//		    printf("Enter the value of no.:");
//		    scanf("%d",&num);
		    isfact(num);

	}

	public static void isfact(int num)
	{
	    int fact=1;
	    int temp=num;
	    while(num>0)
	    {
	        fact=fact*num;
	        num--;

	    }
	    
	   System.out.printf("The factorial of %d is %d",temp,fact);
	}
}
